### ISSUE TITLE
Add Yottaa to an "known incompatible scripts" section in the docs

### ISSUE THREAD

**Issue by ersinakinci** [BuilderIO/partytown#248]
**Is your feature request related to a problem? Please describe.**
I've found through extensive testing that including [Yottaa](https://www.yottaa.com/) on a page--whether it's run using Partytown or not--causes issues with at least Klaviyo and TikTok executing within Partytown, if not more.

When Yottaa is enabled and third-party pixel-type scripts dynamically inject child scripts, those child scripts get `type="text/partytown"`, but then they never execute, or only after a very long timeout. I don't know why this happens.

**Describe the solution you'd like**
Let's add a section to the docs listing "known incompatible scripts" and include Yottaa. We can add to this section over time as new incompatibilities are discovered. It should be made clear which scripts aren't executable using Partytown vs. which break Partytown completely merely by including them on the page.

**Describe alternatives you've considered**
In an ideal world, it would be wonderful if the team could figure out a way to make Yottaa work with Partytown. However, given that Yottaa is a script optimization solution, I suspect that it's using some kind of magic that's fundamentally incompatible with Partytown's own magic, and since it's closed source, we'll never know the root cause of the conflict.

**Comment by adamdbradley** [comment]
Closing this issue since it has become stale, or has been addressed in recent releases and docs updates. If you feel this is still an issue please reopen a new issue. Thanks.
